build/cloudbuild_tools.yaml

```diff
@@ -13,7 +13,7 @@
 #  limitations under the License.
 
 substitutions:
-  _KUBERNETES_VERSION: 1.28.3
+  _KUBERNETES_VERSION: 1.29.0
```

Comment:
we cannot bump the k8s until controller-runtime and controller-tools have an version using k8s 1.29 released